librz/util/subprocess.c

```diff
@@ -714,6 +724,8 @@ struct rz_subprocess_t {
 	int ret;
 	RzStrBuf out;
 	RzStrBuf err;
+	int master_fd; ///< Needed to check whether PTY
```

Comment:
Maybe better to rename to something more neutral?

Reply:
I agree about it not being neutral, but it is the standard term used for it. It will be much easier to understand if I also use the same term that the POSIX documentation uses.